### PR TITLE
Removing auth

### PR DESCRIPTION
**Pull request by vikasvmads**

### CODE REVIEW

File: src/base/components/widgets/Login.js [hunk 1]
```diff
@@ -1,70 +1,70 @@
-import React from 'react';
-import Details from './Details.js';
-import { getServiceUrl, getDomain } from '../../servicelocator.js';
-import { Dropdown, MenuItem, Glyphicon } from 'react-bootstrap';
+// import React from 'react';
```
Comment (AlexeyAnshakov) [inline src/base/components/widgets/Login.js]: I think we need it

File: src/base/components/widgets/Login.js [hunk 2]
```diff
@@ -1,70 +1,70 @@
...
+// import Details from './Details.js';
+// import { getServiceUrl, getDomain } from '../../servicelocator.js';
+// import { Dropdown, MenuItem, Glyphicon } from 'react-bootstrap';
```
Comment (AlexeyAnshakov) [inline src/base/components/widgets/Login.js]: We need it

File: src/base/components/widgets/Login.js [hunk 3]
```diff
@@ -1,70 +1,70 @@
...
+//       {
+//         !!readItLater.length && (readItLater.map((o, index) =>
+//           <MenuItem href={o.url} key={index}>
+//             <i className="material-icons dp_small with_text">bookmark</i>{o.name}
+//           </MenuItem>
+//         ))
+//       }
+//       {!!readItLater.length && (<MenuItem divider />)}
+//       <MenuItem eventKey="1" href="https://core.wrioos.com/create">
+//         <i className="material-icons dp_small with_text">create</i>Create new article
```
Comment (AlexeyAnshakov) [inline src/base/components/widgets/Login.js]: We need it, see prev PR for the comment.

File: src/devserver/index.js [hunk 4]
```diff
@@ -24,7 +25,7 @@ pingerService.use(proxy('https://pinger.wrioos.com/'));
 
 const coreService = express();
 
-coreService.get('/create', (request, response) => {
```
Comment (AlexeyAnshakov) [inline src/devserver/index.js:27]: comment it but don't delete

File: src/iframes/Pinger/js/auth.js [hunk 5]
```diff
@@ -1,31 +1,31 @@
-/**
- * Created by michbil on 05.03.17.
- */
+// /**
```
Comment (AlexeyAnshakov) [inline src/iframes/Pinger/js/auth.js]: use /* and */ to comment a block.

**Review by AlexeyAnshakov — CHANGES_REQUESTED** [review]

File: src/base/components/widgets/Login.js (hunk 3 above)
Comment (vikasvmads) [reply src/base/components/widgets/Login.js]: ok sure

**Pull request merged**